lib/attribute.ts

```diff
@@ -27,13 +28,16 @@ type ValueTypes = Attribute.DataTypeMap[Attribute.DataType];
 type Value = [ValueTypes, Attribute.DataType];
 
 export class Attribute {
-  constructor(attributes: onnx.IAttributeProto[]|null|undefined) {
+  constructor(attributes: onnx.IAttributeProto[]|null|undefined|onnxruntime.experimental.fbs.Attribute[]) {
     this._attributes = new Map();
     if (attributes !== null && attributes !== undefined) {
       for (const attr of attributes) {
-        this._attributes.set(attr.name!, [Attribute.getValue(attr), Attribute.getType(attr)]);
+        if (attr instanceof onnx.AttributeProto) {
+          this._attributes.set(attr.name, [Attribute.getValue(attr), Attribute.getType(attr)]);
```

Comment:
null check is gone here. is it not required?

Reply:
It's not required as we have checked that `attr` is an instance of `onnx.AttributeProto`.